lib/ProductOpener/Tags.pm

```diff
@@ -4937,6 +4937,66 @@ sub cmp_taxonomy_tags_alphabetically ($tagtype, $target_lc, $a, $b) {
 		cmp($translations_to{$tagtype}{$b}{$target_lc} || $translations_to{$tagtype}{$b}{"xx"} || $b);
 }
 
+
+=head2 get_knowledge_content ($tag_type, $tag, $lang_id, $country_code)
```

Comment:
Could you rename $lang_id and $country_code to $target_lc and $target_cc? That's what we use elsewhere in the code for attributes, knowledge panels etc. to specify which language and country is desired.

Reply:
and $tag_type -> $tagtype, $tag -> $tagid (just to be consistent with the rest of the code)

Reply:
Sure, I'm making the changes